dune_client/models.py

```diff
@@ -176,6 +177,18 @@ def from_dict(cls, data: dict[str, Any]) -> ResultMetadata:
 MetaData = Dict[str, Union[int, List[str]]]
 
 
+@dataclass
+class ExecutionResultCSV:
+    """
+    Representation of a raw `result` in CSV format
+    this payload can be passed directly to
+        csv.reader(data) or
+        pandas.from_csv(data)
+    """
+
+    data: BytesIO  # includes all CSV rows, including the header row.
```

Comment:
Does this essentially mean that users will have to parse bytes? I guess the point is to immediately construct a dataframe.

Reply:
yes, this is to avoid the overheads of parsing and type casting to python dicts (if a user wants that, use the normal json output).